### ISSUE TITLE
Bug in parsing module type

### ISSUE THREAD

**Issue by zxqfox** [jsdoctypeparser/jsdoctypeparser#36]
`{module:stream.Readable}`
actual:

``` json
{
  "type": "MODULE",
  "path": "stream.Readable"
}
```

expected (something like):

``` json
{
  "type": "INNER_MEMBER",
  "owner": {
    "type": "MODULE",
    "path": "stream"
  },
  "name": "Readable",
  "meta": { "delimiter": "." }
}
```

Proof: http://usejsdoc.org/tags-module.html (last examples)

**Comment by Kuniwak** [comment]
Hmm, it is a parsing bug about operator precedence (#34).
Now, I have no ideas to resolve this.

@zxqfox Do you have any ideas?

**Comment by zxqfox** [comment]
All we need to catch all cases, and it will be just a task to cover them all or find a good strategy.

**Comment by Kuniwak** [comment]
This problem was fixed on v2.0.0-alpha-6.
You can try it on http://kuniwak.github.io/jsdoctypeparser/ .

**Comment by brettz9** [comment]
If this was merged--and I don't see a tag anymore of `v2.0.0-alpha-6` anymore, is this issue resolved so it can be closed?

**Comment by brettz9** [comment]
Also, when you have a chance, @Kuniwak , could you also close out #32 and #35 if they are merged as well as appears to be the case, and add checkboxes for any items not yet implemented for #24?
